man/rgbgfx.1

```diff
@@ -164,6 +165,37 @@ for a list of formats and their descriptions.
 .It Fl d Ar depth , Fl \-depth Ar depth
 Set the bit depth of the output tile data, in bits per pixel (bpp), either 1 or 2 (the default).
 This changes how tile data is output, and the maximum number of colors per palette (2 and 4 respectively).
+.It Fl i Ar input_tiles , Fl \-input-tileset Ar input_tiles
+Read tiles that will be used to convert this image, and that will always be given the first IDs.
+.Ar input_tiles
+must contain tile data in
+.Dq raw
+format, as generated through the
+.Fl o
+option.
+.Pp
+If used together with
+.Fl o ,
+then the image can contain tiles not in
+.Ar input_tiles .
+Otherwise, the image must be able to be generated using
+.Em only
+the tiles from
+.Ar input_tiles ,
+and thus generate different tile data.
```

Comment:
```suggestion
If the
.Fl o
option is also specified, then the input tiles will be assigned the first tile IDs, and any tiles from the input image that are not in the input tileset will be assigned subsequent IDs.
If no
.Fl o
option is specified, then the tile map can
.Em only
use tiles from the input tileset.
```

Reply:
The wording is overall much better, thank you! I however disagree about “the tilemap can only use [...]”, see one of the comments above.

Reply:
If it's not the case that without `-o` the tile map can only use `-i`'s tiles, then I don't understand this entire sentence. What *is* the difference between providing `-o` or not when `-i` is provided?

Reply:
Ah, I misread “if no” as “if the”, my bad. I would suggest each sentence begin with a different structure to lessen confusion, but I can't come up with such a wording myself, so it's no big deal if we don't.

Reply:
How about "If the `-o` option is also specified..." and "But if the `-o` option is *not* specified..."